custom/icds_reports/utils/aggregation_helpers/agg_ccs_record.py

```diff
@@ -66,7 +66,7 @@ def aggregation_query(self):
             ('anemic_severe', 'sum(ucr.anemic_severe)'),
             ('anemic_unknown', 'sum(ucr.anemic_unknown)'),
             ('extra_meal', 'sum(ucr.extra_meal)'),
-            ('resting_during_pregnancy', 'sum(ucr.resting_during_pregnancy)'),
```

Comment:
It looks like these are changing a lot of values in the output to 0. Why is that?

Reply:
yes, this is because  the output data was contradictory with itself. 
This indicator is such that. this cant be 1 if lady is not pregnant. Cases where it is set to 0, lady is not pregnant.  did not wanted to change pregnant flag do made this 0.
@emord

Reply:
Same for Extra meal indicator